src/module/actor/sheets/SR5BaseActorSheet.ts

```diff
@@ -1110,6 +1110,10 @@ export class SR5BaseActorSheet extends ActorSheet {
 
     _filterSkills(data: SR5ActorSheetData, skills: Skills) {
```

Comment:
this could be done by default param values
` _filterSkills(data: SR5ActorSheetData, skills: Skills = {}) {`

Reply:
I change it